CDS/scripts/js_utils/Datastore.js

```diff
@@ -112,19 +112,14 @@ function get(conn,type, key, callback, userdata) { /* callback as function(err,
     if (type !== "meta" && type !== "media" && type !== "tracks") throw "type must be meta, media or track";
     return new Promise(function (fulfill, reject) {
         getSource(type, conn.whoami).then(function (sourceid) {
+            const stmt = db.prepare("select value from " + type + " where key=?");
```

Comment:
there should be no impact, as we are not actually contacting the database here - just precompiling a statement object that is then called in a loop below